examples/memory/redis/README.md

```diff
@@ -52,6 +52,11 @@ docker info
 
 If you have not already done so, follow the instructions in the [Install Guide](../../../docs/source/quick-start/installing.md#install-from-source) to create the development environment and install NeMo Agent toolkit.
 
+To run this example, install the required dependencies by running the following command:
```

Comment:
This works if in a development environment but not if they installed wheels.

I'm not sure what the correct base assumption here should be...

Reply:
We assume for all examples that the user has cloned the repo.